keanu-python/keanu/vertex/label.py

```diff
@@ -9,6 +10,24 @@
 
 class _VertexLabel(JavaObjectWrapper):
 
+    __separator = "."
+
     def __init__(self, name: str):
-        java_object = k.jvm_view().VertexLabel(name)
+        parts = name.split(_VertexLabel.__separator)
+
+        if len(parts) == 1:
+            java_object = k.jvm_view().VertexLabel(name)
+        else:
+            java_object = k.jvm_view().VertexLabel(parts[0], k.to_java_string_array(parts[1:]))
         super(_VertexLabel, self).__init__(java_object)
+
+    def get_name(self) -> str:
+        return self.unwrap().getQualifiedName()
+
+    def __repr__(self) -> str:
+        return self.get_name().split(".").__repr__()
+
+    @staticmethod
+    def create_from_list(*names: str) -> '_VertexLabel':
```

Comment:
What happens when this list is empty?

Reply:
good call - I've made it raise an exception in that case